html/50x.html

```diff
@@ -0,0 +1,38 @@
+<!--
```

Comment:
Can we put this file into a Lua script like `ngx_tpl.lua`? So that there is no need to create an extra directory.